### PR TITLE
Update configure-extension-file-exclusions-microsoft-defender-antivir…

### PR DESCRIPTION
**Pull request by msbemba**
…us.md

Added a important note about wild card

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7765

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/configure-extension-file-exclusions-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -192,6 +192,7 @@ You can use the asterisk `*`, question mark `?`, or environment variables (such
 > There are key limitations and usage scenarios for these wildcards:
 >
 > - Environment variable usage is limited to machine variables and those applicable to processes running as an NT AUTHORITY\SYSTEM account.
+> - You can only use a maximum of 6 wildcards per entry.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-extension-file-exclusions-microsoft-defender-antivirus.md]:
```suggestion
> - You can only use a maximum of six wildcards per entry.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@denisebmsft Please merge this PR. Thanks

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Pull request merged**